src/content/ccip/concepts/manual-execution.mdx

```diff
@@ -0,0 +1,115 @@
+---
+section: ccip
+date: Last Modified
+title: "CCIP Manual Execution"
+whatsnext:
+  { "Manual execution guide": "/ccip/tutorials/manual-execution", "Learn CCIP best practices": "/ccip/best-practices" }
+---
+
+import { Aside, ClickToZoom, CopyText } from "@components"
+
+<Aside type="note" title="Prerequisites">
+  Read the CCIP [Concepts](/ccip/concepts) and [Architecture](/ccip/architecture) pages to understand all the concepts
+  discussed on this page.
+</Aside>
+
+In general, messages are successfully delivered and processed via CCIP, as detailed in the [Architecture](/ccip/architecture) page. CCIP has an built-in gas bumping mechanism called _Smart Execution_ to ensure users’ messages are reliably delivered to the destination blockchain (Read the [CCIP Execution latency](/ccip/architecture#ccip-execution-latency) section to learn more).
+However, some exceptional conditions may require users to manually execute the transaction on the destination blockchain:
+
+     - The receiver contract on the destination blockchain reverted due to an unhandled exception (for example, a logical error).
+     - The receiver contract on the destination blockchain reverted due to the gas limit being insufficient to execute the triggered function (Note: The gas limit value is set in the [extraArgs](/ccip/api-reference/client#evmextraargsv1) param of the message).
+     - The message could not be executed on the destination chain within CCIP’s Smart Execution time window, currently set to 8 hours. This could happen, for example, during extreme network congestion and resulting gas spikes.
```

Comment:
```suggestion
     - The message could not be executed on the destination chain within CCIP’s Smart Execution time window, which is currently set to 8 hours. This could happen, for example, during extreme network congestion and resulting gas spikes.
```